src/lib/components/colors/Header.svelte

```diff
@@ -88,7 +91,11 @@
   data-column="tool"
   data-needs-changes={hasError}
 >
-  <div class="swatch {type}"></div>
+  <div class="swatch {type} {colorInGamut ? 'in-gamut' : 'out-of-gamut'}">
+    <div class="gamut-warning">
```

Comment:
Should this warning be conditional as well @jamesnw? Currently it's always present, even when the warning by the color formats goes away.

![image](https://github.com/user-attachments/assets/b50e4a43-b12c-4d8b-a0ef-b9dbaa5c52d8)

Reply:
Good call- I made that conditional.